.travis.yml

```diff
@@ -72,164 +82,229 @@ jobs:
     env:
       ANSIBLE_VERSION: "27"
       TESTS_TYPE: unit
+    name: unit tests, Ansible 2.7, Python 3.7
```

Comment:
I'd prefer having UI changes in a separate PR. Having a bunch of mixed patches in one PR makes it non-atomic and hard to review as it's fairly complicated to find bits which are worth checking and easy to miss important changes.
So please let's avoid having such white noise in patches. Also, this change is clearly not "fixes #1697 shard travis builds for python2"

Reply:
P.S. I've extracted names into the master

Reply:
Oh, the conflict is not yet resolved